pkg/ebpftracer/c/tracee.bpf.c

```diff
@@ -1917,7 +1917,7 @@ statfunc u32 cgroup_skb_generic(struct __sk_buff *ctx, enum flow_direction flow_
         return 1;
 
     net_task_context_t *netctx =
-        bpf_sk_storage_get(&net_taskctx_map, sk, 0, 0); // obtain event context
+        bpf_sk_storage_get(&net_taskctx_map, sk, NULL, 0); // obtain event context
```

Comment:
This is just a styling change but I can revert it if there is no consensus. I think it's clearer if the value is explicitly set to the `bpf_sk_storage_get` docs:

https://docs.ebpf.io/linux/helper-function/bpf_sk_storage_get
```
If *value* is **NULL**, the new bpf-local-storage will be zero initialized.
```